samples/ObjectTracker/cs/ObjectTrackerSample_UWP/MainPage.xaml.cs

```diff
@@ -0,0 +1,874 @@
+                        {
+                            await m_bindings[i].SetInputImageAsync(frame);
+                            m_skillEvalStopWatch.Restart();
+                            await m_skill.EvaluateAsync(m_bindings[i]);
+                            m_skillEvalStopWatch.Stop();
+                            evalTicks += m_skillEvalStopWatch.ElapsedTicks;
+                            
+                            // Add result to history
+                            m_trackerHistory[i].Add(
+                                new TrackerResult()
+                                {
+                                    boundingRect = m_bindings[i].BoundingRect,
+                                    succeeded = m_bindings[i].Succeeded
+                                }
+                            );
+                        }
+
+                        // Render results
+                        await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
+                        {
+                            m_renderStopWatch.Restart();
+                            m_objectTrackRenderer.ClearCanvas();
+                            m_objectTrackRenderer.RenderTrackerResults(m_trackerHistory, true);
+                            m_renderStopWatch.Stop();
+
+                            // Print performance measurements
+                            PerformanceBlock.Text = String.Format(
+                                "Skill Eval: {0:0.000}ms, Render: {1:0.000}ms",
+                                evalTicks / (Stopwatch.Frequency / 1000F),
+                                m_renderStopWatch.ElapsedTicks / (Stopwatch.Frequency / 1000F));
+                        });
+                    }
+
+                    // Initialize any new trackers
+                    if (m_drawnRects.Count > 0)
+                    {
+                        // Initialize new trackers if desired
+                        m_bboxLock.Wait();
+
+                        for (int i = 0; i < m_drawnRects.Count; i++)
+                        {
+                            ObjectTrackerBinding binding = await m_skill.CreateSkillBindingAsync() as ObjectTrackerBinding;
+                            await m_skill.InitializeTrackerAsync(binding, frame, m_drawnRects[i]);
+                            m_bindings.Add(binding);
+
+                            // Add corresponding tracker history
+                            m_trackerHistory.Add(
+                                new List<TrackerResult> {
+                                    new TrackerResult()
+                                    {
+                                        boundingRect = binding.BoundingRect,
+                                        succeeded = true
+                                    }
+                                }
+                            );
+                        }
+
+                        m_drawnRects.Clear();
+
+                        m_bboxLock.Release();

```

Comment:
Consider refactoring into separate `RunSkillAsync` method and update README accordingly